ansible/roles/base/tasks/redhat_tasks.yml

```diff
@@ -30,14 +30,15 @@
 - name: install and start ntp
   shell: systemctl enable ntpd
 
-- name: install python-crypto
-  yum: name=python-crypto state=present
-  register: python_crypto_result
+- name: install ansible
+  yum: name=ansible state=present
+  register: ansible_result
   ignore_errors: yes
 
 - name: remove python crypt egg file to work-around https://bugs.centos.org/view.php?id=9896&nbn=2
-  shell: rm -rf /usr/lib64/python2.7/site-packages/pycrypto-2.6.1-py2.7.egg-info
-  when: python_crypto_result.msg == "Error unpacking rpm package python-crypto-2.6.1-1.el7.centos.x86_64\n"
+  shell: rm -rf /usr/lib64/python2.7/site-packages/pycrypto-*.egg-info
```

Comment:
@erikh, 
the change looks fine. But I am just curious about making the string match more generic, did you hit this issue with a different version of python-crypto package? Since this was a workaround I had intentionally kept it for a very specific version.